src/main/java/com/es/phoneshop/utils/ErrorHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.es.phoneshop.utils;
+
+import com.es.phoneshop.exception.OutOfStockException;
+
+import java.util.Map;
+
+public class ErrorHandler {
+    public static String handleError(Exception e) {
+        switch (e.getClass().getName()) {
+            case "java.text.ParseException":
```

Comment:
Please use class.getName() in case values as well.

Reply:
"Constant expression required". I can't use it in cases

Reply:
Yep, and that's why 'switch-case' blocks often aren't welcome. There are ways to do same thing without 'switch-case'

Reply:
Ok, we can leave it as a recommendation. No need to fix this for approve